src/OpenConext/ProfileBundle/Resources/views/MyServices/overview.html.twig

```diff
@@ -67,7 +67,7 @@
                                 </tr>
                                 </thead>
                                 <tbody>
-                                {% for attribute in app.user.attributes %}
+                                {% for attribute in consent.attributes %}
```

Comment:
If `getAttributes()` had a has-er and would throw when attributes=null, we'd get a LogicException here, which'd be good.